cirq/google/calibration/engine_simulator.py

```diff
@@ -364,25 +367,35 @@ def get_calibrations(
 
         return results
 
-    def create_gate_with_drift(self, a: Qid, b: Qid, gate: FSimGate) -> PhasedFSimGate:
+    def create_gate_with_drift(
+        self, a: Qid, b: Qid, gate_calibration: FSimGateCalibration
+    ) -> PhasedFSimGate:
         """Generates a gate with drift for a given gate.
 
         Args:
             a: The first qubit.
             b: The second qubit.
-            gate: Gate which a modified version of should be generated.
+            gate_calibration: Reference gate together with a phase information.
 
         Returns:
             A modified gate that includes the drifts induced by internal state of the simulator.
         """
+        gate = gate_calibration.engine_gate
         if (a, b, gate) in self._drifted_parameters:
             parameters = self._drifted_parameters[(a, b, gate)]
         elif (b, a, gate) in self._drifted_parameters:
             parameters = self._drifted_parameters[(b, a, gate)].parameters_for_qubits_swapped()
         else:
             parameters = self._drift_generator(a, b, gate)
             self._drifted_parameters[(a, b, gate)] = parameters
-        return PhasedFSimGate(**parameters.asdict())
+
+        return PhasedFSimGate(
+            theta=parameters.theta,
+            zeta=parameters.zeta,
+            chi=parameters.chi - 2 * np.pi * gate_calibration.phase_exponent,
```

Comment:
why do you need `phase_exponent` to tag alongside `gate`? Why not use `gate.exponent`?